tests/dummy/app/templates/nodes/detail/draft-registrations/detail.hbs

```diff
@@ -0,0 +1,71 @@
+
+<h2> {{model.draft.registrationSchema.name}} </h2>
+<p> Initiated by: {{model.draft.initiator.fullName}}</p>
+<p> Started: {{moment-format model.draft.datetimeInitiated}} </p>
+<p> Last Updated: {{moment-format model.draft.datetimeUpdated}} </p>
+
+<hr>
+
+<h3> Edit Draft Registration </h3>
+{{#each model.draft.registrationSchema.schema.pages as |page|}}
+
+    {{#each page.questions as |question|}}
+        <p> <label> {{question.title}} </label> </p>
+        {{#if question.description}}
+            <p> {{question.description}} </p>
+        {{/if}}
+        {{#if (eq question.type "choose")}}
+            {{#unless question.description}}
+                <p> <em> Please choose </em> </p>
+            {{/unless}}
+            <form>
+                {{#each question.options as |option|}}
+                    {{#if (eq (checkDataType option) "string")}}
+                        <p> <input onchange={{action "buildForm" value="target"}} type="radio" name={{question.qid}} value={{option}} checked={{selectedHelper option (draftTextInput model.draft.registrationMetadata question)}}> {{option}} </p>
```

Comment:
Right, but you should still be able to use the dash-cased form. That's what we do with the `elem-id` helper, and it didn't seem to require any special config to make it work.

(if ember really is supporting two names at once, that's admittedly very confusing!)